py-polars/polars/expr/string.py

```diff
@@ -2400,9 +2401,9 @@ def contains_any(
         self, patterns: IntoExpr, *, ascii_case_insensitive: bool = False
     ) -> Expr:
         """
-        Use the aho-corasick algorithm to find matches.
+        Use the Aho-Corasick algorithm to find matches.
 
-        This version determines if any of the patterns find a match.
+        Determines if any of the patterns are contained in the string.
```

Comment:
It is not clear what is being compared against "this version". Also updated to be a bit more explicit.